<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 42, 37);
background-color: rgb(43, 43, 43);

color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="insert_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 71);
color: rgb(255, 255, 255);
border-radius: 15px;

font: 15pt &quot;Yu Gothic UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>461</width>
      <height>380</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_minus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_num_1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_num_5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);
background-color: rgb(71, 16, 98);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btn_Mminus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 23, 97);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>MR-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btn_del">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_num_9">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_mult">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_plus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_num_8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_num_3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn_num_0">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="btn_MR">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 23, 97);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>MR</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btn_equall">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btn_div">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="btn_mc">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 23, 97);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>MC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btn_negative">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btn_clear">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_num_7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_num_6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);
background-color: rgb(71, 16, 98);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btn_help">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>HELP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btn_Mplus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 23, 97);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>M+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btn_dot">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 4, 136);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_num_2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btn_M">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_num_4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 16, 98);
border-radius: 15px;
font: 12pt &quot;Yu Gothic UI&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
